### PR TITLE
`treehouses services turtleblocksjs`

### PR DESCRIPTION
**Pull request by vmnet8**
please test install and run
./cli services turtleblocksjs install
./cli services turtleblocksjs up
to go to your browser
yourlocal ip:31001,
then you will see the beautiful turtle!

### CODE REVIEW

**Comment by LordJashin32** [comment]
Had to fix some spacing, etc issues. Sorry. Justin.

File: services/install-turtleblocksjs.sh [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+function install {
+  # create service directory
+  mkdir -p /srv/turtleblocksjs
+
+  # create yml(s)
+  {
+    echo "version: '3'"
+    echo "services:"
+    echo "  turtleblocksjs:"
+    echo "    image: treehouses/turtleblocksjs:balena"
```
Comment (dogi) [inline services/install-turtleblocksjs.sh:12]: balena is not a good tag ...

**Comment by dogi** [comment]
also `package.json` keywords should be enhanced ;)

File: services/install-turtleblocksjs.sh (hunk 1 above)
Comment (vmnet8) [reply services/install-turtleblocksjs.sh:12]: what tag will be better?
am I supposed to add comments about arm6 and arm7 on it?

Comment (vmnet8) [reply services/install-turtleblocksjs.sh:12]:
```
root@lilyxxx0w:~/cli# ./cli.sh services available
couchdb
kolibri
mariadb
mastodon
moodle
netdata
nextcloud
ntopng
pihole
planet
portainer
privatebin
seafile
turtleblocksjs
```
for the pi0w (arm6), services available works.

Comment (vmnet8) [reply services/install-turtleblocksjs.sh:12]: also works on arm7
```
pi@treehouses:~/git_repo/cli $ sudo ./cli.sh services available
couchdb
kolibri
mariadb
mastodon
moodle
netdata
nextcloud
ntopng
pihole
planet
portainer
privatebin
seafile
turtleblocksjs
```

**Pull request merged**